ci/config-algolia-replicas.sh

```diff
@@ -0,0 +1,9 @@
+# Uses dir passed as argument or default one
+backupPath=${1:-"./ci"}
+
+# Replace PLACEHOLDERS with variable available on the CI
+find $backupPath \
```

Comment:
What does this do? It seems like it backs up replica data

Reply:
This script replaces the string `ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA` with the value of the var `$ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA`

Reply:
So we are mutating this file as part of the build process?

Reply:
Can't we just specify the index name on the algolia CLI using -n? I would prefer not to generate new config as part of the build process